application/utils/exception_handler.py

```diff
@@ -0,0 +1,56 @@
+# Utility script to handle exceptions and log output to txt files for debugging
+# purposes.
+
+import inspect
+import logging
+import sys
+import traceback
+
+# Might move in future. Custom exception for bad function call.
+class NonCallableFunctionException(Exception):
+    pass
+
+def try_exec(func, *args):
+    """
+    Executes the passed function within a try-except block.
+    If an exception occurs, the error message is handled and 
+    logged to output files for debugging.
+    Ex: try_exec(sms_service.send_message, list(CAMERAS.keys())[idx], current_time, FEED_URL, filepath)
+
+    :param func: function to execute within try-except.
+    :param *args: arguments to pass when executing func.
+    :return: a tuple with return code and results from func.
+    """
```

Comment:
good comment! Will be super useful :D

Reply:
Might want to change the function in the example later on 😄

Reply:
Facts!